modules/apps/site/site-cms-site-initializer/src/main/java/com/liferay/site/cms/site/initializer/internal/frontend/data/set/filter/AuthorSelectionFDSFilter.java

```diff
@@ -45,23 +63,102 @@ public String getId() {
 	}
 
 	@Override
-	public String getItemKey() {
-		return "id";
+	public String getLabel() {
+		return "author";
 	}
 
 	@Override
-	public String getItemLabel() {
-		return "name";
-	}
+	public List<SelectionFDSFilterItem> getSelectionFDSFilterItems(
+		Locale locale) {
 
-	@Override
-	public String getLabel() {
-		return "author";
+		List<SelectionFDSFilterItem> selectionFDSFilterItems =
+			new ArrayList<>();
+
+		long[] depotEntryGroupIds = _getDepotEntryGroupIds();
+
+		Set<User> userSet = new HashSet<>(_getUsers(depotEntryGroupIds, null));
+
+		userSet.addAll(_getUsers(null, _getUserGroupIds(depotEntryGroupIds)));
+
+		if (userSet.isEmpty()) {
+			return selectionFDSFilterItems;
+		}
+
+		List<User> userList = new ArrayList<>(userSet);
+
+		userList.sort(Comparator.comparing(User::getFullName));
```

Comment:
Instead of creating a HashSet and then copying to a list+sort, use a TreeSet.

Reply:
Using TreeSet now. Thanks!